### PR TITLE
Fix seed

### PR DESCRIPTION
**Pull request by AyanorII**

### CODE REVIEW

**Comment by policiacaro** [comment]
looks fantastic, thanks for cleaning up all the empty lines/comments

**Comment by AyanorII** [comment]
thanks, I'll wait Nicole's approval to merge 👍

File: app/views/owner/planets/index.html.erb [hunk 1]
```diff
@@ -10,3 +10,11 @@
   <% end %>
 </ul>
 <%= link_to "Add a planet", new_planet_path %>
+<% if policy(@planet).update? %>
```
Comment (nogawa11) [inline app/views/owner/planets/index.html.erb:13]: I think this is the same as the previous request, but you can delete lines 13-20 here

Comment (nogawa11) [reply app/views/owner/planets/index.html.erb:13]: Other than that, every thing looks great!

**Pull request merged**